content/documentation/figure-images.md

```diff
@@ -156,6 +156,60 @@ Quire supports video embeds from either YouTube (`media_type: youtube`) or Vimeo
 
 {{< /q-class >}}
 
+## Add Tables
+
+Zoomable tables can be embedded in your publication as HTML files. In the `static/image/` directory of your project create a new folder called `tables`. Create an .html file in that folder that follows the structure below:
+
+``` go
```

Comment:
Should probably be `html` instead of `go` to ensure proper syntax highlighting

Reply:
Is this what you mean?

<img width="634" alt="Screen Shot 2022-01-25 at 12 22 01 PM" src="https://user-images.githubusercontent.com/62075223/151053544-93ece788-761e-430f-927f-cfa49d5ac39e.png">

Reply:
yep!